### ISSUE TITLE
[2012.04193] Robustness of Accuracy Metric and its Inspirations in Learning with Noisy Labels

### ISSUE THREAD

**Issue by riron1206** [riron1206/read_paper#43]
## 概要
https://arxiv.org/abs/2012.04193
ノイズ多いvalidation set信頼できるか調べた論文

train/validation setにノイズ多いデータセットではval_accを最大化することで最適なモデル(ノイズなしデータセットでval_accを最大化したモデル)が取得できるみたい
ノイズ多い場合ModelCheckPointはval_lossではなくval_accで取った方が良さそう

蒸留（教師モデルの推論結果をtrain setのラベルにして生徒モデルを学習する）でノイズなしのtest setのaccuracyを1-2%改善できる
教師モデル(NT)のアキーテクチャはval_accを最大化した Wide_ResNet-28-10
生徒モデル(NS)のアキーテクチャも同じ Wide_ResNet-28-10 使っている（生徒モデルは小さくしない）

![tbl](https://github.com/chenpf1025/robustness_accuracy/raw/master/table.png)

## コメント
- 公式のgithub: 
    - https://github.com/chenpf1025/RobustnessAccuracy (pytorch)
